test/io_tests/prog_test.py

```diff
@@ -0,0 +1,50 @@
+"""Unit test for KNX/IP gateway scanner."""
```

Comment:
We should not sleep in tests. We can use the time_travel fixture to simulate a sleep